<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN24003207" failures="2" tests="10" name="Test" time="0.107" errors="0" timestamp="2023-06-12T17:29:18 IST">
  <testcase classname="tests.ThirdTest" name="Test1" time="0.017">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="tests.ThirdTest" name="Test1" time="0.002">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="tests.ThirdTest" name="Test1" time="0.002">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="tests.ThirdTest" name="Test1" time="0.002">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="tests.ThirdTest" name="Test1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.ThirdTest.Test1(ThirdTest.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="tests.ThirdTest" name="Test2" time="0.001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="tests.ThirdTest" name="Test2" time="0.001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="tests.ThirdTest" name="Test2" time="0.001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="tests.ThirdTest" name="Test2" time="0.002">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase classname="tests.ThirdTest" name="Test2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.ThirdTest.Test2(ThirdTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test -->
